code/datums/components/fantasy/suffixes.dm

```diff
@@ -47,6 +48,7 @@
 
 //////////// Good suffixes
 /datum/fantasy_affix/bane
+	name = "bane of some random mob"
```

Comment:
I think I'd prefer these names to more accurately reflect the names they apply to the item.

`of <mobtype> slaying (random species, carbon or simple animal)`